RealEstate/apps/core/models.py

```diff
@@ -399,10 +412,18 @@ class User(AbstractBaseUser, PermissionsMixin):
             'unique': ("A user with this email already "
                        "exists.")
         })
-    first_name = models.CharField(max_length=30, default="First",
-                                  verbose_name="First Name")
-    last_name = models.CharField(max_length=30, default="Last",
-                                 verbose_name="Last Name")
+    first_name = models.CharField(max_length=30, verbose_name="First Name")
+    last_name = models.CharField(max_length=30, verbose_name="Last Name")
+    phone = models.CharField(max_length=20, blank=True,
+                             validators=[
+                                 RegexValidator(
+                                     regex="^[0-9-()+]{10,20}$",
+                                     message=("Digits or ()+- only, "
+                                              "minimum length 10"),
```

Comment:
Trivial, but I don't love this message. Can we make it more user-focused? "Please enter a valid phone number."